package.json

```diff
@@ -20,12 +20,16 @@
     "prepublish": "grunt test"
   },
   "devDependencies": {
+    "async": "^0.9.0",
```

Comment:
What is this for? Similarly for all new dependencies:
async, chalk, fs-extra, glob